portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,6 +38,12 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Comment");
+    PreparedQuery results = datastore.prepare(query);
+    for (Entity entity : results.asIterable()) {
+        myComments.add((String) entity.getProperty("text"));
```

Comment:
Huh - I guess we can't comment on existing code in Github, but where did how did `MyComments` on line 37 get introduced. It's not in the parent and it's not in this PR...

Reply:
yup it's a weird artifact that was inherited from a really old commit. can address!